tools/sense_studio/utils.py

```diff
@@ -90,17 +90,19 @@ def get_class_labels(path):
     return config['classes'].keys()
 
 
-def get_gpu_status(path):
+def get_toggle_button_status(path, button_name):
```

Comment:
I would propose calling this `get_project_setting` instead, because toggle button is very UI-related.